.github/workflows/wheezy_tests.yml

```diff
@@ -0,0 +1,51 @@
+name: Wheezy CI
+on:
+  pull_request:
+    paths:
+      - "Formula/patchelf.rb"
+      - "Formula/binutils.rb"
+jobs:
+  wheezy_tests:
+    if: github.repository == 'Homebrew/homebrew-core' && github.event_name == 'pull_request' && ! contains(github.event.pull_request.labels.*.name, 'CI-syntax-only')
```

Comment:
```suggestion
    if: ! contains(github.event.pull_request.labels.*.name, 'CI-syntax-only')
```

Reply:
We have `- if: github.repository == 'Homebrew/homebrew-core'` in `tests.yml` to avoid running CI in forks.
And we also have `github.event_name == 'pull_request'` in the tests.yml file to avoid re-running the build part when merging to master.